libPatches.py

```diff
@@ -0,0 +1,585 @@
+def get_lib_patches():
```

Comment:
this file should probably just be a `.json` file somewhere in `static/mojang` instead

Reply:
We would have to re-serialize the json into an object then parse the object so I would rather keep it this way

Reply:
Ideally we would have a model for this patches format. That way it would trivial to handle these patches